configuration/pih/reports/sql/covid19encounterexport.sql

```diff
@@ -6,98 +6,1328 @@
+    discharge_comments      TEXT
 );
 
 ## POPULATE WITH BASE DATA FROM ENCOUNTER, PATIENT, AND PERSON
 ## EXCLUDING VOIDED, AND INCLUDING ONLY THE RELEVANT ENCOUNTER TYPES
 
-insert into temp_encounter (
+INSERT INTO temp_encounter (
     encounter_id,
     patient_id,
     gender,
     birthdate,
     encounter_type,
     encounter_datetime
 )
-select
+SELECT
     e.encounter_id,
     e.patient_id,
     pr.gender,
     pr.birthdate,
     et.name,
-    e.encounter_datetime
-from
+    DATE(e.encounter_datetime)
+FROM
     encounter e
-        inner join patient p on p.patient_id = e.patient_id
-        inner join person pr on pr.person_id = e.patient_id
-        left join encounter_type et on et.encounter_type_id = e.encounter_type
-where
-    pr.voided = 0 and
-    p.voided = 0 and
-    e.voided = 0 and
-    et.name in ('COVID-19 Admission', 'COVID-19 Progress', 'COVID-19 Discharge');
-
-## REMOVE TEST PATIENTS
-
-delete
-from temp_encounter
-where patient_id in
-      (
-          select a.person_id
-          from person_attribute a
-                   inner join person_attribute_type t on a.person_attribute_type_id = t.person_attribute_type_id
-          where a.value = 'true'
-            and t.name = 'Test Patient'
-      );
-
-# ADD DETAILS FOR PATIENT
-
-update temp_encounter set dossier_num = dosId(patient_id);
-update temp_encounter set zlemr_id = zlemr(patient_id);
-update temp_encounter set address = person_address(patient_id);
-update temp_encounter set phone_number = person_attribute_value(patient_id, 'Telephone Number');
-
-# ADD DETAILS FOR ENCOUNTER
-
-update temp_encounter set encounter_provider = provider(encounter_id);
-update temp_encounter set encounter_location = encounter_location_name(encounter_id);
-
-# ADD OBSERVATIONS
-## TODO
-
-# EXECUTE SELECT TO EXPORT TABLE CONTENTS
-
-SELECT e.encounter_id,
-       e.patient_id,
-       e.dossier_num       as dossierId,
-       e.zlemr_id          as zlemr,
-       e.gender,
-       e.birthdate,
-       e.address,
-       e.phone_number,
-       e.encounter_type,
-       e.encounter_location,
-       e.encounter_datetime,
-       e.encounter_provider
-FROM temp_encounter e
-;
+        INNER JOIN patient p ON p.patient_id = e.patient_id
+        INNER JOIN person pr ON pr.person_id = e.patient_id
+        LEFT JOIN encounter_type et ON et.encounter_type_id = e.encounter_type
+WHERE
+    DATE(e.encounter_datetime) >=  @startDate AND
+    DATE(e.encounter_datetime) <=  @endDate AND
+    pr.voided = 0 AND
+    p.voided = 0 AND
+    e.voided = 0 AND
+    et.name IN ('COVID-19 Admission', 'COVID-19 Progress', 'COVID-19 Discharge');
+
+DELETE FROM temp_encounter 
+WHERE
+    patient_id IN (SELECT 
+        a.person_id
+    FROM
+        person_attribute a
+            INNER JOIN
+        person_attribute_type t ON a.person_attribute_type_id = t.person_attribute_type_id
+    
+    WHERE
+        a.value = 'true'
+        AND t.name = 'Test Patient');
+
+UPDATE temp_encounter 
```

Comment:
I personally much prefer the formatting where these update statements are on a single line.  If find it _much_ easier to read.